cmd/helm/list.go

```diff
@@ -145,28 +147,51 @@ type releaseListWriter struct {
 	releases []releaseElement
 }
 
-func newReleaseListWriter(releases []*release.Release) *releaseListWriter {
+func newReleaseListWriter(releases []*release.Release, formatTime bool) *releaseListWriter {
 	// Initialize the array so no results returns an empty array instead of null
 	elements := make([]releaseElement, 0, len(releases))
 	for _, r := range releases {
-		element := releaseElement{
-			Name:       r.Name,
-			Namespace:  r.Namespace,
-			Revision:   strconv.Itoa(r.Version),
-			Status:     r.Info.Status.String(),
-			Chart:      fmt.Sprintf("%s-%s", r.Chart.Metadata.Name, r.Chart.Metadata.Version),
-			AppVersion: r.Chart.Metadata.AppVersion,
-		}
-		t := "-"
-		if tspb := r.Info.LastDeployed; !tspb.IsZero() {
-			t = tspb.String()
+		var element releaseElement
+
+		if formatTime {
+			element = timeFormattedElement(r)
```

Comment:
Why is it that one place calls out to a function to create the releaseElement while the next location does it inline in this function? The difference between the two instances of releaseElement is the `Updated` portion. Couldn't the releaseElement be created and then the `Updated` portion be set based on the config?

Reply:
You're right, i'll update this bit to something simpler and create `releaseElement` only once.

Reply:
Done :)

Reply:
Could you please review?